src/main/java/net/openhft/chronicle/wire/ValueOut.java

```diff
@@ -826,6 +823,9 @@ default WireOut untypedObject(@Nullable Object value) {
         }
         if (value instanceof Marshallable)
             return marshallable((Marshallable) value);
+        if (value instanceof WriteBytesMarshallable) {
```

Comment:
minor but consider using consistent `if` pattern i.e. line above does not have opening/closing braces for single-line `if`